hack/test-cmd.sh

```diff
@@ -327,13 +327,19 @@ for test in "${tests[@]}"; do
   echo
   echo "++ ${test}"
   name=$(basename ${test} .sh)
+  namespace="cmd-${name}"
 
+  os::test::junit::declare_suite_start "cmd/namespace-setup"
   # switch back to a standard identity. This prevents individual tests from changing contexts and messing up other tests
-  oc project ${CLUSTER_ADMIN_CONTEXT}
-  oc new-project "cmd-${name}"
+  VERBOSE=true os::cmd::expect_success "oc project ${CLUSTER_ADMIN_CONTEXT}"
```

Comment:
Not sure we really need or want this verbosity as it's in the log if someone wants to look, but to keep this PR from changing too much I've kept it verbose